app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%= render partial: "layouts/head" %>
+  </head>
+  <body class="app-theme">
+    <a class="skip sr sr-focusable" href="#main-menu">Skip to main menu</a>
+    <a class="skip sr sr-focusable" href="#basic-search">Skip to search form</a>
+    <% if current_page?(results_path) %>
+      <a class="skip sr sr-focusable" href="#results">Skip to results</a>
+      <% if @filters.present? %>
+        <a class="skip sr sr-focusable" href="#filters">Skip to search filters</a>
+      <% end %>
+    <% end %>
+    <% if controller.controller_name == 'record' && controller.action_name == 'view' %>
+      <a class="skip sr sr-focusable" href="#full-record">Skip to metadata</a>
+    <% end %>
```

Comment:
Are we confident we aren't making things worse?

https://webaim.org/techniques/skipnav/#multiple

Reply:
Can we ever be confident we aren't making things worse? 🙂 

That WebAim article is actually what brought me to the conclusion that multiple skip links are warranted for TIMDEX UI. Skip links to the results and filtersare necessary at minimum, I think, because tabbing through the form can be a really onerous process if geo search or advanced search (or, worse, both) are enabled.

I am much less sure about the skips to nav and to full record metadata, since both of those are near the top of the tab order (and 'skip to meadata' could just be 'skip to main content'). If you feel similarly, then I think this would be a good opportunity to follow up with Darcy for further clarification on why they're needed.

Reply:
Thanks for your added perspective on how we got to this PR.

I still need to dig in a bit to better understand what options are doing here as so far I've just looked at code and didn't really get a feel for what was happening.